ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -87,6 +87,34 @@ daemons:
     enabled: <bool>
 ```
 
+## Event Handlers
+
+### Variables
+
+| Variable | Type | Required | Default | Value Restrictions | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| [<samp>event_handlers</samp>](## "event_handlers") | List, items: Dictionary |  |  |  | Event Handlers |
+| [<samp>&nbsp;&nbsp;- name</samp>](## "event_handlers.[].name") | String |  |  |  | Event Handler Name |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;action_type</samp>](## "event_handlers.[].action_type") | String |  |  | Valid Values:<br>- bash<br>- increment<br>- log | Action Type<br>Type of action<br> |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;action</samp>](## "event_handlers.[].action") | String |  |  |  | Command to execute<br> |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;delay</samp>](## "event_handlers.[].delay") | Integer |  |  |  | Event-handler delay in seconds<br> |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;trigger</samp>](## "event_handlers.[].trigger") | String |  |  |  | Configure event trigger condition. Only supports on-logging<br> |
```

Comment:
Shouldn't trigger also have a value restriction here if we only support on-logging?

Reply:
Is that true, though? Why do we have such statement in current documentation "Only supports on-logging"?
The template itself accepts any value.
Reading the documentation, I don't see why we wouldn't support `on-boot` for example...
https://www.arista.com/en/um-eos/eos-command-line-interface-cli?searchword=event-handler

Reply:
In discussion with @ClausHolbechArista , we will adhere to the current documentation, and I will add this check.